src/components/DefinitionList/DefinitionList.module.css

```
@@ -6,7 +6,7 @@
 /**
  * 1) Definition list (`dl`) that contains a definition term (`dt`) and a definition description (`dd`)
  */
-.definition-list {
+.definition-list { 
```

Extra space snuck in there.
```suggestion
.definition-list {
```